src/student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: false)
+    super
```

Comment:
- [ ] the super method should be called by providing the `age`, `name`  & `parent_permission` parameters. please kindly make this change. the Person class expects especially the age variable because it does not have a default value for it. calling super with these parameters will ensure that the `Person` class is instantiated correctly.